src/DocumentFormat.OpenXml/Packaging/RelationshipErrorHandler.cs

```diff
@@ -79,7 +79,12 @@ internal void Handle(Package package)
                     {
                         using var stream = part.GetStream(FileMode.Open, FileAccess.Write);
```

Comment:
What about keeping it as it was, but opening it with `FileMode.Truncate` (see https://docs.microsoft.com/en-us/dotnet/api/system.io.filemode?view=net-5.0).

Reply:
Not supported it seems 😢 

```
    using var stream = part.GetStream(FileMode.Truncate, FileAccess.Write);
    doc.Save(stream);
```

```
  Message: 
    System.ArgumentException : Truncate not supported
  Stack Trace: 
    PackagePart.GetStream(FileMode mode, FileAccess access)
    RelationshipErrorHandler.Handle(Package package) line 80
    OpenXmlPackage.Load(Package _package) line 83
    OpenXmlPackage.ctor(PackageLoader& loader, OpenSettings settings) line 53
    SpreadsheetDocument.ctor(PackageLoader& loader, OpenSettings settings) line 70
    SpreadsheetDocument.Open(Stream stream, Boolean isEditable, OpenSettings openSettings) line 298
    DocumentOpenTests.RewriteMalformedUri() line 69
```